cli/pkg/cmd/config/config.go

```diff
@@ -34,20 +36,33 @@ func applyConfig(cmd *cobra.Command, args []string) {
 		console.Exit("Could not find an installation of Airy Core. Get started here https://airy.co/docs/core/getting-started/installation/introduction")
 	}
 
-	if twilioApply(conf, clientset) {
-		fmt.Println("Twilio configuration applied.")
+	var configuredComponents []string
```

Comment:
a map better fit for the code (so we don't loop and can get rid of the found function)